docs/building_on_linux.md

```diff
@@ -19,24 +19,37 @@ See below for distro-specific instructions on how to get everything installed
 in order to build Schism Tracker.
 
 To get the source:
-
-    git clone https://github.com/schismtracker/schismtracker.git
-    cd schismtracker && autoreconf -i
+```sh
+git clone https://github.com/schismtracker/schismtracker.git
+cd schismtracker && autoreconf -i
+```
 
 You can then update your schismtracker source directory by going to this
 schismtracker directory and running:
-
-    git pull
+```sh
+git pull
+```
 
 ## Building Schism Tracker
 
 To build Schism Tracker, you should set up a build-directory. From the
 schismtracker directory:
-
-    mkdir -p build && cd build && ../configure && make
+```sh
+mkdir -p build && cd build && ../configure && \
+make -j $(nproc || sysctl -n hw.ncpu || echo 2)
+```
 
 The resulting binary `schismtracker` is completely self-contained and can be
-copied anywhere you like on the filesystem.
+copied anywhere you like on the filesystem.<br>
+You can specify custom compiler flags, e.g. to optimize schismtracker
+system-dependently:
+```sh
+make clean  # recompiling needed after changing flags
+CFLAGS="-Ofast -march=native" ../configure --enable-extra-opt
+make -j $(nproc || sysctl -n hw.ncpu || echo 2)
+```
+For debugging, you can set the flags to <samp>"-g -Og"</samp>.<br>
+To get more information, run ```../configure --help```.
```

Comment:
The Og flag is ignored,
when passing --enable-profiling to ../configure, the flags are set to -g -O2 -g. Is this supposed to happen? Shouldn't the flags be set to -g -Og when passing this?